vueui/src/components/view/mcomputer/system/config.vue

```diff
@@ -1,19 +1,24 @@
 <template>
   <div>
-    <i>Welcome to computer configuration utility. Please consult your system administrator if you have any questions about your device.</i><hr>
+    <i>Welcome to the NanoTrasen™ computer configuration utility.
+      Please consult your system administrator if you have any questions about your device.</i><hr>
     <h2>Power Supply</h2>
     <vui-item label="Battery Status:">
       <span v-if="battery">Active</span>
       <span v-else>Not Available</span>
     </vui-item>
-    <vui-item v-if="battery" label="Battery Rating:">{{battery.rating}}</vui-item>
+    <vui-item v-if="battery" label="Battery Rating:">{{battery.rating}} Wh</vui-item>
```

Comment:
Aren't these ratings in mWh according to your changes?

Reply:
I think the standard is still that power_usage is watts, so since I'm changing it properly now to be per second, watt-hours is fine. I want to go through and do a proper overhaul of power at some point anyway to actually make units displayed uniformly everywhere, since it's a huge mess right now.